cmd/earthly/helper/autocomplete.go

```diff
@@ -23,31 +23,38 @@ import (
 
 // to enable autocomplete, enter
 // complete -o nospace -C "/path/to/earthly" earthly
+//
+// alternatively, you can run earthly with COMP_LINE and COMP_POINT set; for example:
+// COMP_LINE="earthly ./buildkitd+buildkitd --" COMP_POINT="32" ./build/linux/amd64/earthly
+// COMP_LINE="earthly ~/test/simple+test -" COMP_POINT="28" ./build/linux/amd64/earthly
 func AutoComplete(ctx context.Context, cli *base.CLI) {
 	_, found := os.LookupEnv("COMP_LINE")
 	if !found {
 		return
 	}
 
-	cli.SetConsole(cli.Console().WithLogLevel(conslogging.Silent))
-
-	err := autoCompleteImp(ctx, cli)
-	if err != nil {
-		errToLog := err
+	_, debugEnabled := os.LookupEnv("EARTHLY_AUTOCOMPLETE_DEBUG")
```

Comment:
> Can also use a CLI flag

This isn't applicable. This code is only executed when determining auto-complete values, which is called by bash or zsh. Autocompletion is executed with `COMP_LINE` and `COMP_POINT` envs, using another env to trigger debugging logs feels appropriate.

Reply:
The name, `EARTHLY_`, seems to fit with the others in `flags/global.go`. And the `cli` arg is already supplied to this function. On its own, one more `LookupEnv` isn't a big deal, but they tend to proliferate over time. Up to you.

Reply:
It's not really meant for most end users; but rather just a hidden mode that we can ask users to enable via an export when they report autocompletion bugs. I think the major differentiation is that since it will never be used as a `--flag`, it doesn't really belong in the `flags` package.

Reply:
That's fair. LGTM.